### DISCUSSION TITLE
Passing array of ID(s) in fieldId param in Matrix Block queries does not return custom field content

### DISCUSSION THREAD

**Discussion by chasegiunta** [craftcms/cms#8764] [Ideas]
### Description
In Craft 2, I'm 90% sure you could pass multiple fieldId's in Matrix Block queries to return their custom field content.

In Craft 3, while you can return custom field content with a sole field ID, placing that field ID in an array, or passing multiple within an array, will not return the custom field content.

Is this functionality being removed intentional?

### Steps to reproduce

1.
   ```php
   $matrixFieldOne = Craft::$app->fields->getFieldByHandle('matrixOne')->id;
   $blocks = MatrixBlock::find()
       ->type('blockTypeA')
       ->fieldId([$matrixFieldOne])
       ->all();
   ```

2. Loop $blocks, attempt to grab a plaintext field content, observe _*nothing*_ ;-)

3.
   ```php
   $matrixFieldOne = Craft::$app->fields->getFieldByHandle('matrixOne')->id;
   $blocks = MatrixBlock::find()
       ->type('blockTypeA')
       ->fieldId($matrixFieldOne)
       ->all();
   ```

4. Loop `$blocks`, attempt to grab a plaintext field content, observe 'dat content.

### Additional info

- Craft version: 3.3.18.4
- PHP version: 7.3

**Comment by brandonkelly** [comment]
This was also the case in Craft 2, and the reason for it is that each Matrix gets its own content table, and element queries aren’t currently able to pull in content from multiple content tables at the same time.

I will leave this open though, as this is something we hope to make possible in Craft 4.

**Comment by chasegiunta** [comment]
~ _walks back that whole **90% sure** claim_ ~
